pypika/tests/test_selects.py

```diff
@@ -157,6 +157,22 @@ def test_select__nested_subquery(self):
                          'FROM (SELECT "sq0"."foo","sq0"."bar" '
                          'FROM (SELECT * FROM "abc") "sq0") "sq1") "sq2"', str(q))
 
+    def test_correlated_subquery(self):
+        subquery = Query.from_(
+            self.table_abc, correlated_table=self.table_efg
```

Comment:
I'm wondering if we can make this a bit more implicit, as it is in SQL, although that's tricky because python is always going to  interpret the subquery expression first. Perhaps we could just remove validation on the table entirely?